manubot/cite/csl_item.py

```diff
@@ -1,6 +1,77 @@
 from manubot.cite.citekey import standardize_citekey, infer_citekey_prefix, is_valid_citekey
 
 
+csl_item_type_fixer = {
+    'journal-article': 'article-journal',
+    'book-chapter': 'chapter',
+    'posted-content': 'manuscript',
+    'proceedings-article': 'paper-conference',
+    'standard': 'entry',
+    'reference-entry': 'entry',
+}
```

Comment:
Just to check - is "citeproc" effectively a large "csl_item processor"? Does that mean merging `citeproc.py` and `csl_item.py` is a possibility because they operate on csl item? As of now I see a dictinction that `csl_item.py` and `CSL_Item` are useful extensions of dict class and `citeproc.py` holds more complex csl item validators.

Reply:
I think another point of distinction between methods and functions is use of keyword parameters for behaviour switches: for functions this is more natural than for class methods, which tend to focus on one just a single responsibility, and generally something rather small.